src/main/resources/db/migration/changes/0007_CreateSchoolsTable.json

```diff
@@ -0,0 +1,65 @@
+{ "databaseChangeLog": [
+    {
+        "changeSet": {
+          "id": "changeset-0007",
+          "author": "pconrad",
```

Comment:
author should really be `ZhenyuYu1` but ok.

Reply:
Done